### ISSUE TITLE
Hide old message-metadata

### ISSUE THREAD

**Issue by JustinB555** [ShoyuVanilla/FoundryVTT-Chat-Portrait#83]
I was wondering if there can be a be small style change to hide the message-metadata once the roll has been collected? It just feels odd to see the same type of message twice, and it conflicts with some UI modules
![Hide Old Message_Metadata](https://user-images.githubusercontent.com/60529781/146343588-50ccd12a-49c2-4ece-aad8-efe1a72ad763.png)
.

**Comment by p4535992** [comment]
Can't you just disable the module setting "Adds a textual tag to chat messages" for that ?

**Comment by JustinB555** [comment]
Haha, okay okay, I found what the issue was. So to note: when you use Chat Portrait with DnD5e UI, DnD5e UI adds a message indicator similar to what you add. I had thought that it was a default card thing that Foundry did; my mistake sorry. Now, this doesn't solve anything base (as it is still connected to a module that I want to use), but I am completely fine with diving into their CSS to change it.

TLDR;
Chat Portrait and DnD5e UI have some slight UI conflicts related to textual tags.
